tasks/rosa/hosted-cp/rosa-hcp-provision/0.2/rosa-hcp-provision.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rosa-hcp-provision
+spec:
+  description: |
```

Comment:
Should we include in the description of the steps this task performs?

Reply:
good point, done